src/api/annotation/AnnotationAPI.js

```diff
@@ -107,6 +108,10 @@ export default class AnnotationAPI extends EventEmitter {
             throw new Error(`At least one target is required to create an annotation`);
         }
 
+        if (!Object.keys(targetDomainObjects).length) {
```

Comment:
Because the documentation above is a lie and I need to fix it. In fact, `targetDomainObjects` is an object as we need the targets to be unique based on their keyStrings. I'll update the documentation.